charts/cc-schema-registry-proxy/README.md

```diff
@@ -0,0 +1,36 @@
+
+
+# cc-schema-registry-proxy
+
+![Version: 0.1.1](https://img.shields.io/badge/Version-0.1.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0](https://img.shields.io/badge/AppVersion-1.0-informational?style=flat-square)
+
+A Helm chart for Confluent Cloud schema registry proxy. This proxy service is used when RADAR-base platform is used with Confluent Cloud based schema registry. It forwards requests to schema registry with an additonal basic authentication header with Confluent Cloud schema registry credentials. This service will be enabled if `cc.enabled = true`.
```

Comment:
@K1Hyve Please check if this is good enough to explain what it does.

Reply:
Yes I didn't noticed it before.